workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/sales-enablement/fragments/search-results-page/index.html

```diff
@@ -0,0 +1,8 @@
+<div class="container-fluid container-fluid-max-xl d-flex flex-row flex-wrap align-items-start search-page">
+	<div class="search-page__filters col-12">
+		<lfr-drop-zone id="searchFilters"></lfr-drop-zone>
+	</div>
+	<div class="search-page__results col">
+		<lfr-drop-zone id="searchResults"></lfr-drop-zone>
```

Comment:
Let's use kebab-case for html ids and classes as best practices.
change it to `search-results`